SimPEG/PF/Gravity.py

```diff
@@ -27,16 +27,16 @@ def __init__(self, mesh, **kwargs):
     def fwr_op(self):
         # Add forward function
         # kappa = self.model.kappa TODO
-        rho = self.rhoMap*self.model
+        rho = self.rhoMap * self.model
```

Comment:
here if you call `rho = self.rho` that will do the same thing (and work in the case where users are not doing an inversion and only provide `rho` as an input to the problem